pkgs/applications/editors/emacs/elisp-packages/manual-packages/gn-mode/package.nix

```diff
@@ -0,0 +1,18 @@
+{
+  gn,
+  melpaBuild,
+}:
+
+melpaBuild {
+  pname = "gn-mode";
+  inherit (gn) src version;
+
+  files = ''("misc/emacs/gn-mode.el")'';
+
+  ignoreCompilationError = false;
+
+  meta = {
+    inherit (gn.meta) homepage license maintainers;
```

Comment:
I do not think it is a good idea to add `gn` maintainers to `gn-mode` because it is possible that only some of them use Emacs and want to maintain `gn-mode`.

In addition, @rennsax, would you like to become a maintainer of `gn-mode`?

Reply:
> I do not think it is a good idea to add `gn` maintainers to `gn-mode` because it is possible that only some of them use Emacs and want to maintain `gn-mode`.
> 
> In addition, @rennsax, would you like to become a maintainer of `gn-mode`?

Oh, I'd like to. I use gn-mode for an essential project and I use Emacs almost every. Shall I add my information in [maintainers/maintainer-list.nix](https://github.com/NixOS/nixpkgs/blob/979cdae832d37a2eb08c150b8188da1731e219f8/maintainers/maintainer-list.nix)?

Reply:
Yes, please open a new PR for that.  Add your entry to that file in the first commit and then add your maintainer id to gn-mode in the second commit. 

You can ping me to merge that PR .